_posts/2020/2020-12-02-load-testing-as-development-tool.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Load testing for measuring impact
+date: 2020-11-26
+header_image: public/load-testing-as-development-tool.jpg
+header_position: bottom
+header_overlay: true
+category: tech-stories
+tags: ["api", "programming", "load test", "artillery", "vegeta"]
+authors: ["Roberto"]
+about_authors: ["rwelzel"]
+---
+
+## Change is written upon all things
```

Comment:
Usually we do not start the post with a heading. I would recommend to remove it.